### DISCUSSION TITLE
Authentication -  which cookies?

### DISCUSSION THREAD

**Discussion by skirk-mpr** [Tyrrrz/YoutubeExplode#766] [Q&A]
When reviewing Developer tools, it seems that there are a lot of cookies set for YouTube - I'd be happy to PR some documentation updates to the Authentication section if someone could tell me which cookies are required?  

I did find some response to other questions which listed some but it may be outdated (e.g. it references  VISITOR_INFO but I only see VISITOR_INFO1_LIVE in Developer tools).

https://github.com/Tyrrrz/YoutubeExplode/issues/376#issuecomment-627324714

```csharp
	var cookies = new List<Cookie>()
	{
		new("YSC","**VALUE**", "/", ".youtube.com"),
		new("PREF","**VALUE**", "/", ".youtube.com"),
		new("VISITOR_INFO1_LIVE","**VALUE**", "/", ".youtube.com"),
		new("SID","**VALUE**", "/", ".youtube.com"),
		new("HSID","**VALUE**", "/", ".youtube.com"),
		new("SSID","**VALUE**", "/", ".youtube.com"),
		new("APISID","**VALUE**", "/", ".youtube.com"),
		new("SAPISID","**VALUE**", "/", ".youtube.com"),
		new("LOGIN_INFO","**VALUE**", "/", ".youtube.com"),
	}.AsReadOnly();

	var client = new YoutubeClient(cookies);

```

When trying the above (but with the actual cookie values ) to access one of my private playlists with
`Playlists.GetVideosAsync("XXXX")`  I get a `PlaylistUnavailableException` which I assuming means, I'm not authenticated with the client to reach the non-public playlist.

**Comment by Tyrrrz** [comment]
I know for a fact that `APISID`/`SAPISID` are required, but not sure if the others are not. You can see here how they're used: https://github.com/Tyrrrz/YoutubeExplode/blob/3118f02d2ad6e458e29c7a2e7abe88ec14af9109/YoutubeExplode/YoutubeHttpHandler.cs#L46-L70
